src/Provider/AbstractProvider.php

```diff
@@ -31,7 +31,9 @@
 
     public $responseType = 'json';
 
-    public $headers = null;
+    public $headers = [];
+
+    public $authorizationHeader;
```

Comment:
I see. If a custom provider continues to use the access token as we've been using it, then nothing is affected; they won't need to mess with `$authorizationHeader` unless they want to.